### ISSUE TITLE
Add new employees to Maximo (Anthony, Eric, Rafael)

### ISSUE THREAD

**Issue by atdservicebot** [cityofaustin/atd-data-tech#17213]
> What application are you using?

Maximo

> Describe the problem.

Need to add new employees to Maximo
- [ ] Anthony Paul W.
- [ ] Eric N.
- [ ] Rafael B. Jr

> How soon do you need this?

Soon — Within a week 

> Requested By
Lionel C.

Request ID: DTS24-114710

**Comment by ReneVarg25** [comment]
### 5/10/24
Reached out to requestor for additional information to confirm users. Could not verify on HR Knack application.

**Comment by ReneVarg25** [comment]
### 5/13/2024
Requestor currently OOO, placing on hold until return.

**Comment by adry-martinez** [comment]
@ReneVarg25 may you please get ahold of Lionel and see if we can get what we need for this issue? Thanks!

**Comment by ReneVarg25** [comment]
### 5/15/2024
Lionel reached out and confirmed the information for Rafael and that Anthony and Eric already have access. Waiting on CTM to complete access.

**Comment by ReneVarg25** [comment]
### 5/17/24
CTM completed account creation. Created labor and updated accesses and account information. Notified Lionel that account was ready for use.
